rules/windows/powershell/powershell_script/posh_ps_set_acl.yml

```diff
@@ -0,0 +1,25 @@
+title: Powershell Set-Acl Windows Folder
+id: cae80281-ef23-44c5-873b-fd48d2666f49
+status: experimental
+description: Detects powershell scripts set ACL to a file in the Windows folder
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/74438b0237d141ee9c99747976447dc884cb1a39/atomics/T1505.005/T1505.005.md
+author: frack113
+date: 2023/07/18
+tags:
+    - attack.defense_evasion
+    - attack.t1222
+logsource:
+    product: windows
+    category: ps_script
+    definition: bade5735-5ab0-4aa7-a642-a11be0e40872
+detection:
+    selection:
+        ScriptBlockText|contains|all:
+            - 'Set-Acl '
+            - '-Path '
+            - 'Windows'
```

Comment:
```suggestion
            - :'\Windows\'
```
just to be sure? I don't see a big downside and a reduce in possible FPs

Reply:
![image](https://github.com/SigmaHQ/sigma/assets/62423083/8e8423c1-f9fc-417e-8cf8-1da62d5c0bba)
Can play with `\` or `/` in powershell on windows 11.
so more a 
```yaml
selection_path:
    - ':\Windows'
    - ':/Windows'
condition all of selection_*
```
What do you think ?

Reply:
Done